tasktiger_admin/templates/tasktiger_admin/tasktiger_task_detail.html

```diff
@@ -36,6 +36,12 @@ <h2><a href="../../../">TaskTiger</a> – <a href="../">{{ queue }} ({{ state }
             <th>Kwargs</th>
             <td>{{ task_data.kwargs }}</td>
         </tr>
+        {% if task.ts %}
```

Comment:
Would a `ts` of `0` be valid and therefore we would want to display that?

Reply:
If score was 0, it wouldn't pass this check anyway
https://github.com/closeio/tasktiger/blob/01085f30b0e39fe3445619e9983c886897989d32/tasktiger/task.py#L452
imo 0 would mean it was never scheduled - probably an invalid state? But not sure.